prana-cli.hcl

```diff
@@ -1,5 +1,5 @@
 description = "prana-cli is the command line client for PranaDB realtime database"
-binaries = ["prana-cli"]
+binaries = ["prana"]
 
 linux {
   source = "https://github.com/cashapp/pranadb/releases/download/v${version}/prana-cli-linux-amd64.gz"
```

Comment:
I think you'll need to rename this?

You want `on unpack { rename { from = "${root}/prana-cli-${os}-${arch}" to = "${root}/prana" } }`

Reply:
Done! Also added the `test` field. Tested locally running ` hermit test prana-cli --trace`